.gitignore

```diff
@@ -8,4 +8,7 @@
 /.vercel_build_output
 .vercel
 /.cache
-/.upm
\ No newline at end of file
+/.upm
+./.config
```

Comment:
you only need the `.config` not `./.config` or `/.config`

Reply:
Ight, i will make fix commit

Reply:
pushed